packages/frontend/package.json

```diff
@@ -47,6 +48,13 @@
         "@tanstack/react-query-devtools": "^4.36.1",
         "@tanstack/react-table": "^8.5.11",
         "@tanstack/react-virtual": "^3.0.0-beta.54",
+        "@tiptap/extension-link": "^2.2.4",
+        "@tiptap/extension-mention": "^2.2.4",
+        "@tiptap/extension-placeholder": "^2.2.4",
+        "@tiptap/pm": "^2.2.4",
+        "@tiptap/react": "^2.2.4",
+        "@tiptap/starter-kit": "^2.2.4",
+        "@tiptap/suggestion": "^2.2.4",
```

Comment:
do we need all these dependencies ?

Reply:
I think we do! 
I'm going to check if we need `@tiptap/pm` one but all the others are required

Reply:
all required